schedule/models/calendars.py

```diff
@@ -182,6 +154,44 @@ def get_absolute_url(self):
         return reverse('calendar_home', kwargs={'calendar_slug': self.slug})
 
 
+class Calendar(AbstractCalendar):
+    """
+    This is for grouping events so that batch relations can be made to all
+    events.  An example would be a project calendar.
+
+    name: the name of the calendar
+    events: all the events contained within the calendar.
+    >>> calendar = Calendar(name = 'Test Calendar')
```

Comment:
`calendar = Calendar(name='Test Calendar')`
Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value. https://www.python.org/dev/peps/pep-0008/#other-recommendations